plugins/occt/module/vtkF3DOCCTReader.cxx

```diff
@@ -651,11 +653,21 @@ int vtkF3DOCCTReader::RequestData(
     STEPCAFControl_Reader reader;
     TransferToDocument(this, reader, doc);
   }
-  else // FILE_FORMAT::IGES
+  else if (this->FileFormat == FILE_FORMAT::IGES)
   {
     IGESCAFControl_Reader reader;
     TransferToDocument(this, reader, doc);
   }
+  else if (this->FileFormat == FILE_FORMAT::XBF)
+  {
+    Handle(TDocStd_Application) app = new TDocStd_Application();
+    BinXCAFDrivers::DefineFormat(app);
```

Comment:
you need to guard this too

Reply:
this one is already inside an `#if F3D_PLUGIN_OCCT_XCAF`, is that good?

Reply:
ha right!